### ISSUE TITLE
Table view seems to have disappeared in 4.2.2.9

### ISSUE THREAD

**Issue by ToonTalk** [jmoenig/Snap#2276]
In 4.2.2.9 I see

![image](https://user-images.githubusercontent.com/2138696/49434801-0334e080-f7ad-11e8-9e35-cfb6a6a341ab.png)

While in 4.2.2.8 I see (as documented in the manual)

![image](https://user-images.githubusercontent.com/2138696/49434874-26f82680-f7ad-11e8-852d-acc9883faca2.png)

**Comment by jguille2** [comment]
Hi @ToonTalk ,

I can't reproduce your issue. For me, it is working fine in 4.2.2.9

Maybe you have disabled the "Table support" option?
![tablesupport](https://user-images.githubusercontent.com/2926315/49438243-19976800-f7be-11e8-873e-1e9fbb13d880.png)

**Comment by ToonTalk** [comment]
Thanks. I definitely didn't change the setting intentionally. 

I just logged off and reloaded Snap! and Table support wasn't selected. But then loaded Snap! into Edge (where I've never logged in) and sure enough Table support was enabled. And I just tried to reload Snap! a second time but with a hard refresh (holding down control) and Table support was enabled.

I typically use a local cached version of Snap! (4.2.2.8 just now) and sometimes use the release and/or dev version. I do notice that sometimes after switching back and forth it losses my "Zoom blocks" setting. Perhaps this switching is sometimes corrupting my settings?
